code/modules/modular_computers/computers/modular_computer/interaction.dm

```diff
@@ -30,6 +30,16 @@
 
 	proc_eject_ai(usr)
 
+/obj/item/modular_computer/verb/eject_personal_ai()
+	set name = "Eject Personal AI"
+	set category = "Object"
+	set src in view(1)
+
+	if(use_check_and_message(usr))
+		return
+
+	proc_eject_personal_ai(usr)
```

Comment:
Don't put proc in a proc's name

Reply:
all the other verb / proc combos have it like that. feel free to change it in a modular computer clean-up PR